credentials/README.md

```
@@ -105,8 +130,87 @@ export BUNDLE_RUBYGEMS__PKG__GITHUB__COM="op://Dev/GITHUB_TOKEN/credential"
     git config --global user.signingkey "$(cat ~/.ssh/id_ed25519.pub)"
 ```
 
+## 1Password Vault 推奨構造
+
+テンプレートが正しく動作するように、1Password の Vault "Dev" に以下の構造でアイテムを作成してください：
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)